### DISCUSSION TITLE
Download instead of assemble

### DISCUSSION THREAD

**Discussion by sualeh** [jreleaser/jreleaser#727] [Ideas]
@aalmiray  - I assemble my project elsewhere, so when I release with JReleaser, my "assemble" step simply becomes a download. It would be good for JReleaser to have a "download" assembler built-in for this use case. What do you think?

**Comment by aalmiray** [comment]
I thought about it after seeing how you assemble the installers in SchemaCrawler.

JReleaser's original goal is to make it easy to release a project, with or without binaries. If binaries are needed then those are the responsibility of the build tool of choice. 

However later we added the option to assemble binaries based on certain conditions, to make it easier for the packagers and releasers. 

This has led to more integrations in the tool that can help you assemble and release on a local environment rather than on CI. 

For this reason an option to download assets required for assembly OR release might be a good idea, as it would also be platform agnostic. Probably after `1.0.0` is released.

**Reply by sualeh** [reply]
Thanks. I have created https://github.com/jreleaser/jreleaser/issues/728 to track this.
